source/backend/dal/Repositories/CompensationRequisitionRepository.cs

```diff
@@ -51,6 +51,8 @@ public PimsCompensationRequisition GetById(long compensationRequisitionId)
                 .Include(c => c.PimsCompReqFinancials)
                     .ThenInclude(y => y.FinancialActivityCode)
                 .Include(x => x.AcquisitionOwner)
+                .Include(x => x.AcquisitionFilePerson)
+                .Include(x => x.InterestHolder)
```

Comment:
Github doesn't let me add comments to code that has not been changed so I will put it here but note I am talking about the next method in this file - **Update(...)**

- Shouldn't this PR also fix updating payee information for an existing compensation requisition?
- I see the TODO: fix this is still there....